lib/api/entities.rb

```diff
@@ -199,6 +199,11 @@ class SSHKey < Grape::Entity
       expose :id, :title, :key, :created_at
     end
 
+    class SSHKeyWithUser < Grape::Entity
```

Comment:
If you do `SSHKeyWithUser < SSHKey`, you don't need to repeat the first line of exposing :)